com/Airport.cpp

```diff
@@ -156,8 +156,10 @@ void Airport::updateTobt(const std::string& callsign, const std::chrono::utc_clo
         root["vacdm"] = Json::Value();
         root["vacdm"]["tobt"] = Airport::timestampToIsoString(tobt);
         root["vacdm"]["tsat"] = Airport::timestampToIsoString(types::defaultTime);
-        if (false == manualTobt)
-            root["vacdm"]["tobt_state"] = "NOW";
+        if (false == manualTobt) {
+            root["vacdm"]["tobt_state"] = "CONFIRMED";
+            it->second[FlightEuroscope].tobt_state = "CONFIRMED"; // should allow for quicker color change in colorize function

```

Comment:
Why is it faster? Shouldn't the sync directly be done in the next cycle?
Looks like a dangerous hack. We should think about a resync of both blocks after every call if it is too slow.

I don't think that it has side effects, but debugging will be faster if an other internal structure is manipulated, if we don't expect it.

Reply:
That change was based on user feedback, which stated that the color changes are to slow. An was an attempt do immediately change the variable hoping to see an faster update. Instead of waiting for the next pull of data.

However we later found out that the colorize functions were slow / unprecise because we casted the difference to minutes instead of seconds. 
As this issue is now correct we can remove it, especially as it only gives (at best) an advantage of the current time between pulls, which should be 2 seconds. 

Please advise how you want to proceed. Removing is fine for me.

Reply:
I suggest to remove it, if the bug is now really fixed.